src/Client/Core/DurableTaskClient.cs

```diff
@@ -368,7 +392,7 @@ public virtual Task ResumeInstanceAsync(string instanceId, CancellationToken can
     /// <see cref="PurgeResult.PurgedInstanceCount"/> indicating the number of orchestration instances that were purged.
     /// </returns>
     public abstract Task<PurgeResult> PurgeAllInstancesAsync(
-        PurgeInstancesFilter filter, CancellationToken cancellation = default);
+        PurgeInstancesFilter filter, bool recursive = true, CancellationToken cancellation = default);
```

Comment:
Same issue with the other comment: adding a net-new abstract method is a breaking change.

Reply:
Changed to virtual